tools/pasta-setup.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+set -e
+
+python create_project.py $2
+./manage.py parsearchive $1 --list-id $2
+./manage.py dumparchive $2
+tar -xf *.tar
```

Comment:
And move the script to patchwork. Again, this has nothing to do with PaStA, it's a patch for patchwork.